sdk/ai/ai-inference-rest/src/modelClient.ts

```diff
@@ -28,7 +28,7 @@ export default function createClient(
   { apiVersion = "2024-05-01-preview", ...options }: ModelClientOptions = {},
 ): ModelClient {
   const endpointUrl = options.endpoint ?? options.baseUrl ?? `${endpointParam}`;
-  const userAgentInfo = `azsdk-js-AiModelInference-rest/1.0.0-beta.2`;
+  const userAgentInfo = `azsdk-js-ai-inference/1.0.0-beta.2`;
```

Comment:
This is auto-gen code? I like the new string, but wondering why it changed

Reply:
it is autogen, yes.